buku

```diff
@@ -983,20 +989,20 @@ class BukuDb:
         resultset = self.cur.fetchall()
         recs = len(resultset)
         if not recs:
-            LOGERR('No matching index or title immutable or empty DB')
+            LOGERR(gettext('No matching index or title immutable or empty DB'))
             return False
 
         # Set up strings to be printed
         if self.colorize:
-            bad_url_str = '\x1b[1mIndex %d: Malformed URL\x1b[0m\n'
-            mime_str = '\x1b[1mIndex %d: HTTP HEAD requested\x1b[0m\n'
-            blank_url_str = '\x1b[1mIndex %d: No title\x1b[0m\n'
-            success_str = 'Title: [%s]\n\x1b[92mIndex %d: updated\x1b[0m\n'
+            bad_url_str = gettext('\x1b[1mIndex %d: Malformed URL\x1b[0m\n')
+            mime_str = gettext('\x1b[1mIndex %d: HTTP HEAD requested\x1b[0m\n')
+            blank_url_str = gettext('\x1b[1mIndex %d: No title\x1b[0m\n')
+            success_str = gettext('Title: [%s]\n\x1b[92mIndex %d: updated\x1b[0m\n')
         else:
-            bad_url_str = 'Index %d: Malformed URL\n'
-            mime_str = 'Index %d: HTTP HEAD requested\n'
-            blank_url_str = 'Index %d: No title\n'
-            success_str = 'Title: [%s]\nIndex %d: updated\n'
+            bad_url_str = gettext('Index %d: Malformed URL\n')
+            mime_str = gettext('Index %d: HTTP HEAD requested\n')
+            blank_url_str = gettext('Index %d: No title\n')
+            success_str = gettext('Title: [%s]\nIndex %d: updated\n')
```

Comment:
These lines contain the same text, the only difference being existence of colour tags. Shouldn't this use the same translation lines, only with/without the colour tags (i.e. `'\x1b[1m%s\x1b[0m\n' % gettext('Index %d: No title')`)?